packages/nestjs/src/index.ts

```
@@ -0,0 +1,318 @@
+    private meta: Record<string, any>,
+    private logger: NestLogger,
+    private isDebug: boolean,
+  ) {}
+  private prepareMessage(args: LazyLoggerMessage[]) {
+    const obj = {
+      ...(this.meta || {}),
+    };
+    const strs: string[] = [];
+    args = args
+      .flatMap((arg) => (typeof arg === 'function' ? arg() : arg))
+      .flat(Infinity);
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)